src/polyswarmd/artifacts/artifacts.py

```diff
@@ -37,11 +50,16 @@ def post_artifacts():
     session = app.config['REQUESTS_SESSION']
 
     # Since we aren't using MAX_CONTENT_LENGTH anymore, we have to check each.
-    files = [(f'{i:06d}', f)
-             for (i, f) in enumerate(request.files.getlist(key='file'))
-             if f.content_length <= g.user.max_artifact_size]
+    try:
+        files = [(f'{i:06d}', f)
```

Comment:
that is cool, didn't know you could do that in an f-string.

Reply:
Pretty nifty, thanks @sbnetops

Reply:
:+1: